addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -67,6 +69,16 @@ export class FloorScreen extends Component {
         }
         await this.pos.unsetTable();
     }
+    get floorBackroundURL() {
+        return this.activeFloor.floor_background_image
+            ? "data:image/png;base64," + this.activeFloor.floor_background_image
+            : false;
+    }
+    get floorBackround() {
+        return this.activeFloor.floor_background_image
+            ? "url(" + this.floorBackroundURL + ")"
+            : "none";
+    }
```

Comment:
Why there is two methods ? 
```suggestion
    get floorBackround() {
        return this.activeFloor.floor_background_image
            ? "data:image/png;base64," + this.activeFloor.floor_background_image
            : "none";
    }
```